core-it-suite/src/test/java/org/apache/maven/it/MavenITmng2339BadProjectInterpolationPre4Test.java

```diff
@@ -0,0 +1,75 @@
+package org.apache.maven.it;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.maven.it.util.ResourceExtractor;
+
+import java.io.File;
+
+/**
+ * This is a test set for <a href="https://issues.apache.org/jira/browse/MNG-2339">MNG-2339</a>.
+ *
+ *
+ */
+public class MavenITmng2339BadProjectInterpolationPre4Test
+    extends AbstractMavenIntegrationTestCase
+{
+    public MavenITmng2339BadProjectInterpolationPre4Test()
+    {
+        super( "(2.0.8,4.0.0-alpha-1-SNAPSHOT)" ); // 2.0.9+
```

Comment:
The `SNAPSHOT` is not necessary. The magic will still work.

Reply:
Of course!